Lexicon.md

```diff
@@ -11,7 +11,9 @@
 * __Observation__: __Information__ that crosses the __Agent Boundary__, entering the __Agent__.
 * __Object__: A set of correlated __Observations__.
 * __System__: An __Object__ composed of correlated member __Objects__.
-* __Model__: A __System__ that represents another __System__. It can be assigned a truth value which expresses the accuracy of the representation. Examples of __Models__ include maps, blueprints, simulations, mathematical models, beliefs and claims.
+* __Model__: A __System__ that represents another __System__. It can be assigned a truth __Value__ which expresses the accuracy of the representation. Examples of __Models__ include maps, blueprints, simulations, mathematical models, beliefs and claims.
```

Comment:
It seems you are conflating to different meanings of "value" here. In this case value is a measure and below it is a preference (which can also be measured of course). Is it worth distinguishing or do you see all values as preferences? I guess I'm thinking truth values like the kinds of values assigned to variables in programs.

Reply:
This is part of the reason that I thought it would be a good idea to nail down what truth is. 

One way that I was trying to reason about this is that a truth value can be considered a preference of accuracy. In other words, in the space of all possible models for a system, I assign more value to the ones I find most accurately representing empirical reality.

But you are right I might be conflating the fact that in mathematics you can say "x has a value of 5" and there is no preference involved. I'm open to changing this.

Reply:
Switched around the wording a bit.